### ISSUE TITLE
resources to add

### ISSUE THREAD

**Issue by ahwagner** [cancervariants/metakb#4]

**Comment by ahwagner** [comment]
We need to describe the resources we plan to add to the meta-knowledgebase.

Somatic-focused interpretation resources:
* Oncokb "Clinical"
* Jax-CKB
* PMKB
* CIViCdb
* MolecularMatch
* CGI
* HemOnc.org ontology

Germline-focused interpretation resources:
* BRCA Exchange
* ClinVar (expert-panel reviewed)

Biological / Oncogenic annotation resources:
* COSMiC
* OncoKB "Biological"
* CGI Catalog of Validated Oncogenic Mutations

**Comment by DavidTamborero** [comment]
CGI Catalog of Validated Oncogenic Mutations was a compillation of available KBs annotating effects 'compatible' with oncogenic (e.g. a BRCA1 reported as cancer-predisposing variant is likely a LoF (i.e. 'oncogenic') also when observed in the somatic line)
So do not use this database as it is not a 'independent' source of data 

Also, I m unsure of COSMIC gathering 'oncogenic' varaints (or 'functional' or 'biological' or whatever label)  but I think that they 'only' gathered a set of biomarkers of drug resistance. Let me know if I'm incorrect, it would be interesting to me to know! 

Finally, note the overlap between ClinVar and BRCA-exch (the latter use the former as one of its sources)

**Comment by ahwagner** [comment]
Thanks @DavidTamborero, will exclude CVOM per your recommendation.

As for COSMIC, one type of data they collect is drug resistance mutations, which would be a useful addition as a "somatic interpretation" resource, albeit they would have to be classified as AMP/ASCO/CAP Tier 3 (unlike the other somatic resources, which are all Tier 1/2), as these annotations–despite being publication linked–lack any indication of evidence level. I think I'll propose to add this as a source.

The data for COSMIC I was referring to here I envisioned fitting in as an observed somatic frequency annotation; something we've had requests to incorporate before. I'll revise to treat this as a distinct entity, so overall it will look as such:

Somatic-focused interpretation resources:
* OncoKB "Clinical"
* Jax-CKB
* PMKB
* CIViCdb
* MolecularMatch
* CGI
* COSMIC "Resistance mutations"
* HemOnc.org ontology

Germline-focused interpretation resources:
* BRCA Exchange
* ClinVar (expert-panel reviewed)

Clinical trial resources:
* Jax-CKB
* MolecularMatch
* Matchminer (maybe? not sure if this is still working its way to public)

Other annotation resources:
* COSMIC (mutation frequency by cancer type)
* OncoKB "Biological"

**Comment by bpitel12** [comment]
I wonder how open the large group would be to tapping into algorithmic models for determining potential oncogenicity for variants that are not included in the harmonized KB.  COSMIC displays HMM predictions.  CGI uses the OncoDrive tools on the backend to make some predictions: https://bbglab.irbbarcelona.org/tag/oncodrive/  Definitely doesn't hold the same weight as published evidence, but might help in interpreting a variant that is not well known.

**Comment by DavidTamborero** [comment]
i strongly advocate to provide (computational) insights for variants not contained in knowledgebases, which is fundamental for e.g. guide allocation of patients to clinical trials (which at least in Europe tends to be the preferred choice as compared to off-label drug use)

In the tools i developed in the past I generally find no objections for that among clinicians as far as everything is well documented and organized (and you re using the analyses that are more appoppriate for each particular context)

Worth to mention here, I ve just finished another reporting system for panel seq data that  is going to be used (starting next week) in a 100% clinical setting, with the two particularities that (a) this will be used by clinicians of different countries (countries in Europe have quite different approaches); and (b) will be used by clinicians that maybe are not that skilled in this kind of tools. So I hope I will learn a lot with that, I will let you know!

**Comment by ahwagner** [comment]
@bpitel12 I agree that an algorithm for determining oncogenicity, even a simple ruleset, is something that we will need to have for this project.

@DavidTamborero mentioned something similar in another issue: https://github.com/cancervariants/metakb/issues/3#issuecomment-443148080

I have opened a separate issue to drive this discussion, #12

**Comment by jeremywarner** [comment]
This could be of interest: https://www.ncbi.nlm.nih.gov/pubmed/29961819

-Jeremy

Jeremy L. Warner MD, MS, FAMIA
Associate Professor of Medicine and Biomedical Informatics at Vanderbilt University
Medical Director, Vanderbilt Cancer Registry & SCT Data Analysis Team
Chair, AMIA Visual Analytics Work Group
Deputy Editor, HemOnc.org

Rhonda N. Fox
Administrative Assistant II
rhonda.fox@vumc.org<mailto:rhonda.fox@vumc.org>
615.936.2587

**Comment by bpitel12** [comment]
Clinical trial resources: consider adding/collaboration with My Cancer Genome - we have representation from their group on the large VICC-VTB call.

**Comment by ahwagner** [comment]
https://moalmanac.org/

**Comment by ahwagner** [comment]
https://cosmic-blog.sanger.ac.uk/new-product-actionability-census/
